packages/react-components/react-tabs/library/etc/react-tabs.api.md

```diff
@@ -109,11 +110,12 @@ export type TabSlots = {
 
 // @public
 export type TabState = ComponentState<TabInternalSlots> & Pick<TabProps, 'value'> & Required<Pick<TabProps, 'disabled'>> & {
-    appearance?: 'transparent' | 'subtle';
+    appearance?: 'transparent' | 'subtle' | 'filled';
     iconOnly: boolean;
     selected: boolean;
     contentReservedSpaceClassName?: string;
     size: 'small' | 'medium' | 'large';
+    shape?: 'circular';
```

Comment:
I was thinking about two options for this - `circular?: boolean` or `shape?: 'circular'`. Currently, the boolean option appears simpler, but using a discriminated union might enhance scalability and align better with other components like Button, which have shape options. 

I'd appreciate the feedback on this and am willing to change this based on it.

Reply:
makes sense to me, maybe somebody else might have different opinion on this